src/server/report/test.js

```diff
@@ -464,8 +330,7 @@ class Test extends LitElement {
 		}
 
 	}
-	_renderTestResults(browser, testData) {
-		const result = testData.results.find(r => r.name === browser);
+	_renderTestResults(result) {
```

Comment:
Seems like this is maybe overkill now, and the `div` could just be added to `_renderTestResult`?

Reply:
I guess there's a lot of early `return`s in there.  Could maybe move this up into:
```
content: html`<div class="result">this._renderTestResults(result)</div>`,
```
Either way, minor, just an idea

Reply:
Yeah I had started leaving space for when this is actually going to be rendering multiple test results -- i.e. once you can click on an entire file and see all its tests. Definitely overkill right now though.